package.json

```diff
@@ -71,6 +71,8 @@
     "devDependencies": {
         "c8": "^10.1.2",
         "cypress": "^13.13.2",
+        "axe-core": "^4.10.0",
+        "cypress-axe": "^1.5.0",
```

Comment:
@mojotalantikite I added these two dependencies here too. Correct me if things changed but, I did this because Cypress in CI runs from root and on local is runs from `services/cypress` without this here CI could not find the dependencies. A lot has changed since so is there a better, but simple, way to not have to install in both places? 

I could do some configuration in each test to point to the location of the install based on deployment, but if installing the deps in both doesn't cost or complicate much would this be ok?

Reply:
@mojotalantikite @JasonLin0991 imo this could alkso be good time to also consider if cypress should be moved considering new world we are. Throwing it out there, not trying to suggest moving the entire folder is needed ...just want us to think expansively since we are also doing a lot around modularizing and simplifying our top level architecture right now so maybe we are seeing another opportunity here.

Reply:
This is totally fine. I personally think it's clearer to declare the dependencies in each service's `package.json` that needs them. Back with Yarn 1 everything got hoisted and when we moved to pnpm it started complaining about missing packages, as it was easy to just ignore the dependency if it was already installed elsewhere with Yarn. 

Plus with pnpm everything is symlinked around, so it's not actually going to install it in two places, it's just going to throw in a symlink from the global package store to each service's `node_modules` that declares it.